src/node_url.cc

```diff
@@ -1196,6 +1197,14 @@ namespace url {
               url->path.push_back(segment);
             }
             buffer.clear();
+            if (url->scheme == "file:" &&
+                (ch == kEOL ||
+                 ch == '?' ||
+                 ch == '#')) {
+              while (url->path.size() > 1 && url->path[0].length() == 0) {
```

Comment:
The updated spec says `greater than 1` instead of `> 0`:
https://github.com/whatwg/url/blob/2f4b7f74533398c41f4316157f027a33a627caf9/url.bs#L2070